website/docs/setup.mdx

```diff
@@ -114,16 +147,50 @@ By default, SDK searches for new translation once per application load, but not
 
     Where `interval` - defines translations update time interval in seconds. Minimum allowed interval is 15 minutes (900 seconds).
 
-## Change locale programmatically
+### Change locale programmatically
 
 By default, the SDK relies on the device locale. To change the SDK target language on the fly regardless of the device locale, use the following method:
 
 ```swift
+import CrowdinSDK
+
 CrowdinSDK.enableSDKLocalization(true, localization: "<language_code>")
 ```
 
 Where `<language_code>` is the target language in [ISO 639-1](http://www.loc.gov/standards/iso639-2/php/English_list.php) format.
 
+You can also use the `CrowdinSDK.currentLocalization` property to get or set the current localization language code used by the Crowdin SDK. When you set this property, the SDK triggers a localization download if needed:
```

Comment:
CrowdinSDK.currentLocalization:
- Main method to switch language.
- When set to some language code - will download and use new localization from Crowdin(if available).
- When set to nil - localization will be auto detected based on available languages in Crowdin and system preferred languages.
- UI won't be refreshed automatically.

Reply:
Updated in #316